modules/serverless-creating-broker-admin.adoc

```diff
@@ -3,36 +3,43 @@
 // * serverless/knative_eventing/serverless-using-brokers.adoc
 
 [id="serverless-creating-broker-admin_{context}"]
-= Creating a broker automatically using namespace annotation
+= Creating a Broker as a cluster administrator using namespace annotation
 
-If you have cluster administrator permissions, you can create a broker automatically by annotating a namespace.
+If you have cluster administrator permissions, you can create the `default` Broker automatically by using namespace annotation.
+
+The examples in this procedure use the `default` namespace.
+You can choose a different namespace if desired, but will need to update the `oc` commands accordingly.
 
 .Prerequisites
 * Knative Eventing installed.
 * Cluster administrator permissions for {product-title}.
 
 .Procedure
 
-. Annotate your namespace by entering the following commands:
+* Annotate your namespace by entering the following commands:
 +
+
 ----
-$ oc label namespace default knative-eventing-injection=enabled <1>
-$ oc -n default get broker default
+$ oc label namespace default knative-eventing-injection=enabled
 ----
-<1>  Replace `default` with the desired namespace.
+
 +
-The line shown in this example will automatically create a broker named `default` in the `default` namespace.
+
+----
+$ oc -n default get broker default
+----
 
 [NOTE]
 ====
 Brokers created due to annotation will not be removed if you remove the annotation. You must manually delete them.
 ====
 
 [id="serverless-deleting-broker-admin_{context}"]
-= Deleting a broker that was created using namespace annotation
+= Deleting a Broker that was created using namespace annotation
```

Comment:
To delete this broker administrator needs to remove the `knative-eventing-injection=enabled` label from namespace. Without it broker will be recreated.

Reply:
You're right. Here's a command:

`oc label namespace default knative-eventing-injection-`